content/posts/why-we-use-cucumber-for-end-to-end-testing/index.md

```diff
@@ -0,0 +1,63 @@
+# Why we use Cucumber for end-to-end testing?
+
+Before knowing why we use cucumber for End-to-end testing. Let understand what is end-to-end testing and cucumber
+
+## What is End to end testing?
+
+End to end testing (E2E testing) is a software testing method that validates entire software from beginning to end.
+
+The purpose of end-to-end testing is to simulate what a real user scenario and testing whole software for dependencies, data integrity and communication with other systems, interfaces, network connectivity and databases to exercise complete production like scenario.
+
+## What is Cucumber?
+
+What is the world said:
+
+> Cucumber is a software tool that supports behavior-driven development (BDD).Central to the Cucumber BDD approach is its ordinary language parser called Gherkin. It allows expected software behaviors to be specified in a logical language that customers can understand. As such, Cucumber allows the execution of feature documentation written in business-facing text. It is often used for testing other software. It runs automated acceptance tests written in a behavior-driven development (BDD) style.
+
+## What is BDD?
+
+
+
+## Why we use Cucumber for end-to-end testing?
```

Comment:
Why you have an empty block here?

Reply:
That's feedback from team in review meeting, I added a title section first. Will push content later

Reply:
Same here https://github.com/manabie-com/manabie-com.github.io/pull/3/files#diff-5dff4770660c24c9ea9596e9c9907d6b5b005833b9642dc46d13ccee295ada9bR62-R63

Reply:
ok